xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -31,7 +31,7 @@ When `Utf8JsonReader` encounters invalid JSON, it throws a <xref:System.Text.Jso
 
 Since this type is a ref struct, it doesn't directly support async. However, it does provide support for reentrancy to read incomplete data and to continue reading once more data is presented.
 
-To be able to set max depth while reading OR allow skipping comments, create an instance of <xref:System.Text.Json.JsonReaderState> and pass it to the reader.
+To be able to set max depth while reading OR allow skipping comments, create an instance of <xref:System.Text.Json.JsonReaderOptions> and pass it to the reader.
```

Comment:
When do we use `<see cref=...` vs `<xref:...`?

Reply:
cc @rpetrusha

Reply:
@ahsonkhan xref is markdown language, so it is used in remarks, cref is used in normal xml tags.

Reply:
Or more precisely, <xref> can only be used in a CDATA section, as on line 24. Otherwise, <cref> should be used.